deployments/README.md

```diff
@@ -0,0 +1,75 @@
+# Bridge Deployments
+
+### General notes
+
+- Substrate authorities are named: `Alice`, `Bob`, `Charlie`, `Dave`, `Eve`, `Ferdie`.
+- Ethereum authorities are named: `Arthur`, `Bertha`, `Carlos`.
+- `Dockerfile`s are designed to build & run nodes & relay by fetching the sources
+  from a git repo.
+
+  You can configure commit hashes using docker build arguments:
+  - `BRIDGE_REPO` - git repository of the bridge node & relay code
+  - `BRIDGE_HASH` - commit hash within that repo (can also be a branch or tag)
+  - `ETHEREUM_REPO` - git repository of the OpenEthereum client
+  - `ETHEREUM_HASH` - commit hash within that repo (can also be a branch or tag)
+  - `PROJECT` - a project to build withing bridges repo (`bridge-node` or `ethereum-poa-relay`
+    currently)
+
+  You can however uncomment `ADD` commands within the docker files to build
+  an image from your local sources.
+
+### Requirements
+
+Make sure to install `docker` and `docker-compose` to be able to run & test
+bridges deployments locally.
+
+### Polkadot.js UI
+
+To teach the UI decode our custom types used in the pallet, go to: `Settings -> Developer`
+and import the [`./types.json`](./types.json)
+
+## TestPoa
+
+`TestPoa` is a test bridge deployment between a test Ethereum PoA network and
+test Substrate network.
+It's main purpose is to make sure that basic PoA<>Substrate bridge operation works.
+The network is being reset every now and then without a warning.
+
+### UI
+
+Use [wss://testpoa.bridges.test-installations.parity.io/](https://polkadot.js.org/apps/)
+as a custom endpoint for [https://polkadot.js.org/apps/](https://polkadot.js.org/apps/).
+
+### How to run locally?
```

Comment:
I think this should come before the UI stuff, probably right after the `Requirements` section